modules/combined/test/tests/reference_residual/reference_residual_perfgraph.i

```diff
@@ -0,0 +1,199 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
```

Comment:
@permcody I think we are using the GlobalParams for displacements after all--if I remove the global params then I need to add the displacements line to the master action block like so:

```
[Modules/TensorMechanics/Master]
 displacements = 'disp_x disp_y disp_z'
  [./all]
    displacements = 'disp_x disp_y disp_z'
    volumetric_locking_correction = true
    incremental = true
    save_in = 'saved_x saved_y saved_z'
    eigenstrain_names = thermal_expansion
    strain = FINITE
    decomposition_method = EigenSolution
  [../]
[]
```